integration-tests/src/test/kotlin/org/stellar/anchor/platform/AnchorPlatformIntegrationTest.kt

```diff
@@ -80,15 +79,22 @@ class AnchorPlatformIntegrationTest {
     @BeforeAll
     @JvmStatic
     fun setup() {
-      platformServerContext =
-        AnchorPlatformServer.start(
-          SEP_SERVER_PORT,
-          "/",
-          mapOf("stellar.anchor.config" to "classpath:/integration-test.anchor-config.yaml"),
-          true
-        )
+      val configMap =
+        mapOf("stellar.anchor.config" to "classpath:/integration-test.anchor-config.yaml")
+
+      platformServerContext = AnchorPlatformServer.start(SEP_SERVER_PORT, "/", configMap, true)
 
-      AnchorReferenceServer.start(REFERENCE_SERVER_PORT, "/")
+      ServiceRunner.startAnchorReferenceServer()
+
+      Thread {
+          StellarObservingService.start(configMap)
```

Comment:
There are two starting patterns here

- `AnchorPlatformServer.start()`
- `StellarObservingService.start()`
- `ServiceRunner.startAnchorReferenceServer()`

Can we unify them? If I can vote, I would prefer the `ServiceRunner.start()` pattern because it is the way that we invoke the services in docker.

Reply:
I'll check if we can use ServiceRunner and still inkject the integration-test.anchor-config.yaml file.

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/546/commits/c60b809bd33af699b4fd14f958c141bc0fd3aab0.